assets/templates/aws.billing/schema-b/gotext.tpl

```diff
@@ -0,0 +1,61 @@
+{{- $currency := generate "aws.billing.currency"}}
```

Comment:
I assume this assigns a variable which later is used 3 times. It it expected that this value is the same for all 3 entries?

Seeing support of variables here. Can variables also be combined, something like:

```
{{- $a := generate "a"}}
{{- $b := generate "b"}}
{{- $c := $a + $b}}
```

(not a feature request, just curious)

Reply:
> I assume this assigns a variable which later is used 3 times

accidental observation:
```
{{ $a := generate "a"}} 1.{{ $a }} 2.{{ $a }} 3.{{ $a }}
```
will produce different output than:
```
1.{{ generate "a" }} 2.{{ generate "a" }} 3.{{ generate "a" }}
```

since `generate` will emit a different value for each call (unless the value is hardcoded or the cardinality is 1)

Reply:
> It it expected that this value is the same for all 3 entries?

An AWS Account is tied to a specific currency, given we collect information on an account basis every item should use the same single currency.

> Can variables also be combined,

That syntax is not supported, but Golang text/templates have [pipelines](https://pkg.go.dev/text/template#hdr-Pipelines), can run [methods/functions](https://pkg.go.dev/text/template#hdr-Functions) and additional templating helpers may be provided, like the [Masterminds/sprig](http://masterminds.github.io/sprig/) library we include.